src/commands/kv/mod.rs

```diff
@@ -104,9 +104,8 @@ fn url_encode_key(key: &str) -> String {
 // For handling cases where the API gateway returns errors via HTTP status codes
 // (no KV error code is given).
 fn give_status_code_context(status_code: StatusCode) {
-    match status_code {
-        StatusCode::PAYLOAD_TOO_LARGE => message::warn("Returned status code 413, Payload Too Large. Make sure your upload is less than 100MB in size"),
-        _ => (),
+    if let StatusCode::PAYLOAD_TOO_LARGE = status_code {
```

Comment:
Can we use `if let` like this? We could honestly just do `if StatusCode::.. = status_code` for now.

Reply:
This was recommended to me by the compiler. Seems to work fine (merged this change with `feat-kv-additions` locally to test)

```console
$ wrangler kv:bucket upload --binding KV ./static
🌀  Parsing favicon.ico...
🌀  Parsing home.html...
🌀  Parsing .DS_Store...
🌀  Parsing amp/750kb.html...
🌀  Parsing amp/2mb.html...
🌀  Parsing amp/blog.html...
🌀  Parsing amp/home.html...
🌀  Parsing amp/.DS_Store...
🌀  Parsing amp/900kb.html...
🌀  Parsing amp/500kb.html...
🌀  Parsing amp/4mb.html...
🌀  Parsing amp/625kb.html...
🌀  Parsing amp/home2.html...
🌀  Parsing amp/1mb.html...
🌀  Parsing amp/825kb.html...
🌀  Parsing amp/flowerblog.html...
🌀  Parsing sitemap.xml...
🌀  Parsing robots.txt...
⚠️  Returned status code 413, Payload Too Large. Make sure your upload is less than 100MB in size
⚠️  Error 10024: could not put value for key "amp/4mb.html": value length of 4002377 exceeds limit of 2097152
🕵️  See documentation
```

Reply:
wait that's not the same path jk ignore this comment